config/schemas/settings.json

```diff
@@ -314,6 +314,10 @@
 				"stickyHeroInfoWindows" : {
 					"type" : "boolean",
 					"default" : true
+				},
+				"enableAutocombatSpells" : {
+					"type": "boolean",
+					"default": true
```

Comment:
I'd keep it off by default - AI is using mana *too* freely right now

Reply:
IMO it is natural to keep it on, so you start with no confusing handicaps on AI side, H3 also uses spells by default